docs/tutorial.md

```diff
@@ -386,7 +597,7 @@ export default {
 - [store.defineTable](api/read-model/store.md#definetable)
 - [store.insert](api/read-model/store.md#insert)
 
-The type of the physical store used to save data is defined by a Read Model connector:
+The type of the physical store used to save data is defined by a Read Model connector. Add the following code to the **config.dev.js** file.
 
 **config.dev.js**
 
```

Comment:
Can you make the following change to line 394 below?

Line 394:
`// The 'config.dev.js' file defines setting used only in the development environment.`

Can  you change "setting" to "settings"?

Reply:
Done

Reply:
Thanks.